languages/en_US.po

```diff
@@ -674,644 +837,780 @@ msgstr ""
 "The ICP license number will be shown at the footer and linked to the website "
 "of <i>Ministry of Industry and Information Technology of the Chinese "
 "government</i>. Leave it blank to disable it. This may be particularly "
-"useful if your website is hosted in China。"
+"useful if your website is hosted within China."
+
+#: admin_fn.php:647
+msgid "在前端载入 jQuery"
+msgstr "Load jQuery on the front end"
+
+#: admin_fn.php:653
+msgid ""
+"自 2.0.0 版本起，MDx 已不再依赖 jQuery 且不再在前端载入 jQuery 脚本。如果你在"
+"升级 MDx 后碰到页面显示不正常等问题，请尝试打开此选项。<br><strong>这是一个临"
+"时选项，将会在未来版本中被移除。如果你的前端页面中有依赖 jQuery 的其他资源，"
+"请确保其不受 MDx 移除 jQuery 的影响。</strong><br>有关更多从 MDx 1.x 升级至 "
+"2.x 的信息，请参阅<a href=\"https://doc.flyhigher.top/mdx/zh-CN/upgrade_tip/"
+"\">主题文档</a>"
+msgstr ""
+"Since version 2.0.0, MDx no longer depends on jQuery and stops loading it on "
+"the front end. If you encounter display issues after upgrading, please try "
+"to enable this option.<br><strong>This is a interim option and will be "
+"removed in the future. If you have other front-end assets that depend on "
+"jQuery, please make sure that they are not affected by this upgrade.</"
+"strong><br>For more information on upgrading from MDx 1.x to 2.x, please "
+"read the <a href=\"https://doc.flyhigher.top/mdx/en-US/upgrade_tip/\">theme "
+"documentation</a>"
```

Comment:
The link seems to be unavailable (for both `zh-CN` and `en-US`)
https://doc.flyhigher.top/mdx/zh-CN/upgrade_tip/

Reply:
Since I'm still testing the theme, the doc has not been updated yet